### DISCUSSION TITLE
Feature:  Save web page(s) as PDF

### DISCUSSION THREAD

**Discussion by forrie** [Kovah/LinkAce#598] [Ideas & Feature Requests]
It would be useful to save the current website page as a PDF, which can be stored in the database.

There's a Mac app that does this, Yojimbo, and it's very handy -- would be a useful feature for LinkAce.

**Comment by BadCo-NZ** [comment]
Second this, [LinkWarden](https://github.com/linkwarden/linkwarden) has this feature.

> Auto-capture a screenshot and PDF from each website.

**Comment by Kovah** [comment]
This will never be added directly in LinkAce. You might connect a private web archive in the future (see #496), but I'm refusing to add any archiving, PDF generation or other full text search to LinkAce.
